apps/showcase/src/components/utilities/date-picker-input/date-picker-input-pres.context.ts

```diff
@@ -7,7 +10,9 @@ import type {
  */
 export interface DatePickerInputPresContextInput {
   /** ID of the html element used for selection */
-  id: string;
+  id: InputSignal<string>;
+  /** Label of the datepicker */
+  label?: InputSignal<string>;
```

Comment:
```suggestion
  label: InputSignal<string>;
```
as you set the input as required
```
public label = input.required<string>();